.eslintrcignore

```diff
@@ -0,0 +1,6 @@
+dist/
+node_modules/
+.nyc_ouput/
+coverage/
+lib/
+environment/
```

Comment:
Nitpick: the standard name of the file is `.eslintignore`, without `rc` in the middle.

Reply:
I've blindly copied it from the other repo. I'll have to fix it there as well